config.m4

```diff
@@ -46,6 +46,11 @@ if test "$PHP_DDTRACE" != "no"; then
     components/string_view/string_view.c \
   "
 
+  ZAI_SOURCES="\
```

Comment:
What is the motivation for having a zai dedicated root folder instead of using the `components` one? Is it to keep `components` PHP-free?

Reply:
Yes, components are being tested on containers that do not have PHP installed and we want to keep them completely decoupled from the Zend Engine.

Also, ZAI is being written as if it was its own open source project completely decoupled from the tracer. The ZAI SAPI will have extensibility baked in so that we can use it to test the tracer separately. We'll be able to add custom INI settings without having to use INI files (something the embed SAPI cannot do) and also provide SAPI-level functions that we can expose to userland for testing purposes only. This also potentially opens up an easier entry point for fuzzing later on.

Reply:
love it ❤️  thanks